cmake/gRPC.cmake

```diff
@@ -74,8 +74,13 @@ if(GWHISPER_FORCE_BUILDING_GRPC OR GRPC_NOT_FOUND)
     endif()
     # Adding the "EXCLUDE_FROM_ALL" here, to prevent install targets to be added
     # As we do not want to intsall whole gRPC and its depdendencies to the system.
-    # NOTE: This unfortunately does not work for tests, which causes us to do some
-    #       Hacks because of zlib tests being not disableable
+    # We do not want to build any tests from grpc
+    set(BUILD_TESTING OFF)
```

Comment:
does this have any effect? I think i tried `BUILD_RESTING OFF` also in this context, but it did not do anything.

Reply:
In an ideal world yes. Because in an ideal world people would call "enable_testing()" iff a flag was set to enable testing.
Zlib module does not live in an ideal world.